### PR TITLE
Basic rest-express integration

### PR DESCRIPTION
**Pull request by pradel**

### CODE REVIEW

File: docs/cookbook/express.md [hunk 1]
```diff
@@ -1 +1,31 @@
 # Express
+
+*JSAccounts adapter for Expressjs*
```
Comment (davidyaha) [inline docs/cookbook/express.md]: JSAccounts middleware for ExpressJS

File: docs/cookbook/express.md [hunk 2]
```diff
@@ -1 +1,31 @@
...
+```javascript
+import express from 'express';
+import bodyParser from 'body-parser';
+import AccountsServer from '@accounts/server';
+import accountsExpress from '@accounts/rest-express';
+
+AccountsServer.config({}); // Config your accounts server
+
+// Config express
+app.use(bodyParser.urlencoded({ extended: true }));
```
Comment (davidyaha) [inline docs/cookbook/express.md]: This is not mandatory as we don't use url encoded params

File: docs/cookbook/express.md [hunk 3]
```diff
@@ -1 +1,31 @@
 # Express
+
+*JSAccounts adapter for Expressjs*
+
+## Install
```
Comment (davidyaha) [inline docs/cookbook/express.md]: Maybe explain before that, that we are adding in cors support inside of the package (although we want to remove it)

Comment (pradel) [inline docs/cookbook/express.md]: Maybe just let it like that for now?

Comment (davidyaha) [inline docs/cookbook/express.md]: Fine by me

**Pull request merged**